testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -35,7 +35,7 @@ fn neon_integration_test_conf() -> (Config, StacksAddress) {
     let keychain = Keychain::default(conf.node.seed.clone());
 
     conf.node.miner = true;
-    conf.node.wait_time_for_microblocks = 500;
+    conf.node.wait_time_for_microblocks = 6000;
```

Comment:
Is this only needed for the microblock integration test? If so, can you just set this wait time in that test?

Reply:
It's also used in the size-check test, but I can move this change to just these two tests.

Reply:
Addressed in ee12847